tests/test_repocard.py

```diff
@@ -94,16 +94,17 @@
 model-index:
 - name: RoBERTa fine-tuned on ReactionGIF
   results:
-  - metrics:
-    - type: accuracy
-      value: 0.2662102282047272
-      name: Accuracy
-    task:
+  - task:
       type: text-classification
       name: Text Classification
     dataset:
       name: ReactionGIF
       type: julien-c/reactiongif
+    metrics:
+    - type: accuracy
+      value: 0.2662102282047272
+      name: Accuracy
+      verified: false
```

Comment:
nit - should we add `config` in example here too, since you added that with this PR?

Reply:
Thanks for this feedback!

In order to do this, were you thinking that I should also update the inputs to metadata_eval_result function here: https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/repocard.py#L80

An additional change like this is needed, because the test here won't pass otherwise:
https://github.com/huggingface/huggingface_hub/blob/main/tests/test_repocard.py#L166

Reply:
I think that's what you were thinking. I just pushed a change where I added optional fields to metadata_eval_result and added documentation. lmk what you think!